src/leihs/my/password_restore/back.clj

```diff
@@ -90,108 +88,118 @@
     (merge (anti-csrf-props request) <>)
     (merge {:footer {:appVersion release/version}} <>)))
 
+(defn user-with-unique-id-new [tx user-unique-id]
+  (let [sql {:select :* :from :users :where [:or [:= :users.org_id user-unique-id]
+                                             [:= :users.login user-unique-id]
+                                             [:= [[[:lower :users/email]]] [:lower (-> user-unique-id (or ""))]]
+                                             ]}
+        query (sql-format sql)]
+    (jdbc/execute-one! tx (spy query))))
```

Comment:
Wurde direkt in my implementiert anstatt die Version von shared-clj (java.jdbc) zu verwenden:
1. https://github.com/leihs/leihs-clj-shared/blob/a5a91b47a4aa3dc1ec134d72dcb79cb035c1fd22/src/leihs/core/sign_in/back.clj#L16
2. https://github.com/leihs/leihs-clj-shared/blob/a5a91b47a4aa3dc1ec134d72dcb79cb035c1fd22/src/leihs/core/sign_in/shared.clj#L45

TODO
- Renaming: user-with-unique-id-new -> user-with-unique-id